IOG-contracts/README.md

```diff
@@ -38,6 +38,31 @@ In the following, we describe the contracts, leaving to the
 `README.md` files in each subdirectory the responsibility to explain how to
 generate UPLC code from their associated contracts.
 
+### Nix-shell
+
+The best way to work with these contracts is to use IOG's Nix development environment from the plutus-apps dependency.
+See [how to build the projects artifacts](https://github.com/input-output-hk/plutus-apps/blob/main/README.adoc#how-to-build-the-projects-artifacts)
+from the plutus-apps readme for information on setting up nix.
+
+With nix set up, you need to enter the nix-shell. There are two ways to do this:
+
+The preferred way is if the `cabal-install` package is installed in your environment:
+```
+$ cabal update
+$ cd dist-newstyle/src/plutus-ap_-348916df41c59834
+$ nix-shell
+$ cd ../../..
```

Comment:
- I do not have `cabal` avaiable outside nix and don't want to install it...
- Isn't it necessary to use `--extra-experimental-features flakes`?
- In which directory are running these commands?

Reply:
1. Instructions to do it without cabal are below this block
2. yes! I'll update the instructions. I have it enabled in my config so I forgot about it.
3. Same directory as the `cabal.project` and this `README.md` file, `IOG-contracts/`

Reply:
Please add 

> Same directory as the cabal.project and this README.md file, IOG-contracts/

this information to the instructions.

Reply:
> I do not have cabal available outside nix and don't want to install it...

I mean, I didn't have to run cabal from the outside of a nix shell, this is what's confusing me.

Reply:
You don't have to use cabal outside of the nix-shell, but it's the best way because it pulls the dependencies into `dist-newstyle`. The `plutus-apps` checkout will be under `dist-newstyle`, and you can run `nix-shell` from there.

Without cabal outside of the nix-shell, you have to manually clone `plutus-apps` to get into the nix-shell, and it will get re-cloned into `dist-newstyle` when you run the cabal commands.